app/helpers/routerHelper.js

```diff
@@ -23,3 +23,29 @@ export const determineContentType = (documentName) => {
   const documentType = documentNameSplit[documentNameSplit.length - 1];
   return `application/${documentType}`;
 };
+
+export const getHealthCheckDependencies = (config) => {
+  const dependencies = [
+    {
+      name: 'Buying Catalogue API',
+      endpoint: `${config.buyingCatalogueApiHost}/health/ready`,
```

Comment:
Use the getEndpoint function once that is merged in

Reply:
have you changed this?

Reply:
yes 👍